eskipfile/watch.go

```diff
@@ -26,9 +27,14 @@ type WatchClient struct {
 
 // Watch creates a route configuration client with file watching. Watch doesn't follow file system nodes, it
 // always reads from the file identified by the initially provided file name.
-func Watch(name string) *WatchClient {
+func Watch(fileName string) *WatchClient {
+	return WatchFile(fileName, eskip.ParseBytes)
+}
+
+func WatchFile(fileName string, parseFunc eskip.ParseFunc) *WatchClient {
```

Comment:
the name of this function should reflect what's the difference from the simple Watch(). My candidate is: WatchParser()